### ISSUE TITLE
I can't log in 

### ISSUE THREAD

**Issue by mskdgmr** [dvx/lofi#319]
Every time I try to log in It takes Me to this Error

![Screenshot 2024-06-09 201844](https://github.com/dvx/lofi/assets/172227422/506a05cf-679b-4869-bde9-55c30e19eb0a)

**Comment by dvx** [comment]
Yeah, it looks like the `auth` endpoint is down on Heroku. Will investigate.
